plugin-examples/src/main/java/com/hotels/styx/ExampleMessageIntercept.java

```diff
@@ -0,0 +1,78 @@
+package com.hotels.styx;
+
+import com.hotels.styx.api.Eventual;
+import com.hotels.styx.api.HttpResponse;
+import com.hotels.styx.api.LiveHttpRequest;
+import com.hotels.styx.api.LiveHttpResponse;
+import com.hotels.styx.api.plugins.spi.Plugin;
+
+import static com.hotels.styx.api.HttpHeaderNames.CONTENT_TYPE;
+import static com.hotels.styx.api.HttpResponseStatus.OK;
+import static java.nio.charset.StandardCharsets.UTF_8;
+
+/**
+ * This shows an example of an inteceptor that responds to a received request
+ */
+
+public class Example implements Plugin {
```

Comment:
Java class names must match the file name. I would not choose either of `Example` (too vague) or `ExampleMessageIntercept` (unclear, not ending with a noun).

Perhaps `EarlyReturnExamplePlugin` would be a good name.

Another thing to note is that there is already a class called `ExamplePlugin`. That was fine when there was only one example plugin, but now that you have added another, it has become too vague. You should come up with a new name and rename it.